src/main/resources/Macros/AWMKanbanMacro.xml

```diff
@@ -371,9 +376,14 @@ ul.card-field-list {
   ## We set as default the original colors: green,blue,orange,yellow,red
   #set($colors = "#8C4,#0AC,#F91,#FC3,#E43")
 #end
+#if($xcontext.macro.params.displayCategories)
+  #set($displayCategories = "$xcontext.macro.params.displayCategories")
+#else
+  #set($displayCategories = "")
+#end
```

Comment:
You can replace this section with:

#set ($displayCategories = "$!xcontext.macro.params.displayCategories")